docs/01-prerequisites.md

```diff
@@ -145,3 +150,5 @@ If at any point you run into issues while exploring OpenShift 4, see the
 
 
 Next: [Installing the Cluster](02-install.md)
+
+[machine-config-daemon-ssh-keys]: https://github.com/openshift/machine-config-operator/blob/master/docs/Update-SSHKeys.md
```

Comment:
Does GH rendering support ... footlinks?

Reply:
I know it's in the markdown syntax but am not sure if GH supports it. If it does we can ignore this comment.

Reply:
GitHub's browser supports this.  Check by clicking "View file" and scrolling down to [here][1].

[1]: https://github.com/openshift/training/blob/7e5f97409a224732c600f871006a424b300257cf/docs/01-prerequisites.md#note